WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -631,7 +632,12 @@ protected void signInError(int messageId, String clientResponse) {
         AnalyticsTracker.track(Stat.LOGIN_FAILED);
         FragmentTransaction ft = getFragmentManager().beginTransaction();
         SignInDialogFragment nuxAlert;
-        if (messageId == org.wordpress.android.R.string.account_two_step_auth_enabled) {
+        if (!NetworkUtils.isNetworkAvailable(getActivity())) {
```

Comment:
> Why not checking this also on the button pressed event?

I wanted to keep all error management in the same place, but you're right, it's better to check that when the sign in button is pressed.